README.rst

```diff
@@ -106,23 +108,26 @@ Folium supports colors and marker icon types (from bootstrap)
 .. code:: python
 
     map_1 = folium.Map(location=[45.372, -121.6972], zoom_start=12,tiles='Stamen Terrain')
-    map_1.simple_marker([45.3288, -121.6625], popup='Mt. Hood Meadows',marker_icon='cloud')
-    map_1.simple_marker([45.3311, -121.7113], popup='Timberline Lodge',marker_color='green')
-    map_1.simple_marker([45.3300, -121.6823], popup='Some Other Location',marker_color='red',marker_icon='info-sign')
+    folium.Marker([45.3288, -121.6625], popup='Mt. Hood Meadows',
+                       icon = folium.Icon(icon = 'cloud')).add_to(map_1)
```

Comment:
Those that are too big we can at least try to indent them in a way that looks "correct."